src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1480,6 +1480,17 @@ class ActorIdStringGenerator : public Generator<std::string> {
     std::string _actorId;
 };
 
+/** `{^NowTimestamp: {}}` */
+class NowTimestampGenerator : public Generator<bsoncxx::types::b_timestamp> {
+public:
+    NowTimestampGenerator(const Node& node, GeneratorArgs generatorArgs) {}
+    bsoncxx::types::b_timestamp evaluate() override {
+        auto now = std::chrono::system_clock::now();
+        uint32_t nowSinceEpoch = std::chrono::duration_cast<std::chrono::seconds>(now.time_since_epoch()).count();
+        return bsoncxx::types::b_timestamp{.increment = 1, .timestamp = nowSinceEpoch};
```

Comment:
I don't fully understand why we have `.increment = 1` here? The sensible default seems to be `0` to me, and if this needs to be configurable it should be an argument to the generator. This might be a `mongod` thing I'm not understanding.